lib/react_on_rails/server_rendering_pool.rb

```diff
@@ -11,6 +11,15 @@ def self.reset_pool
       @js_context_pool = ConnectionPool.new(options) { create_js_context }
     end
 
+    def self.reset_pool_if_server_bundle_was_modified
+      return unless ReactOnRails.configuration.reload_server_js_every_request
+      file_mtime = File.mtime(ReactOnRails.configuration.server_bundle_js_file)
+      @server_bundle_timestamp ||= file_mtime
+      return if @server_bundle_timestamp == file_mtime
+      ReactOnRails::ServerRenderingPool.reset_pool
+      @server_bundle_timestamp = file_mtime
```

Comment:
@justin808 that's where we had it, but if you put it there you're going to be in a new instance every request, so in order to keep the value of the timestamp variable, you'd need to make it a class variable, which is why you had us move it to server_rendering_pool.